dashboard/src/shared/Namespace.ts

```diff
@@ -50,3 +51,18 @@ export default class Namespace {
 export const definedNamespaces = {
   all: "_all",
 };
+
+export function getCurrentNamespace(currentNS: string, availableNS: string[]) {
+  if (currentNS) {
```

Comment:
Don't understand why we even have `currentNS` as an arg - why call this function if we already have the `currentNS`?

Reply:
yes,  it was to avoid doing the check every time I had to call this function.